packages/core-cairo/src/erc721.test.ts

```diff
@@ -5,6 +5,14 @@ import { printContract } from './print';
 
 import { erc721 } from '.';
 
+const allFeaturesON: Partial<ERC721Options> = {
+  mintable: true,
+  burnable: true,
+  enumerable: true,
+  pausable: true,
```

Comment:
Minor suggestion for consistency:
```suggestion
  pausable: true,
  enumerable: true,
```